src/module.c

```diff
@@ -1008,135 +1008,160 @@ int RediSearch_InitModuleInternal(RedisModuleCtx *ctx, RedisModuleString **argv,
 #define INDEX_DOC_CMD_ARGS 2, 2, 1
 #endif
 
-  RM_TRY(RedisModule_CreateCommand, ctx, RS_INDEX_LIST_CMD, IndexList, "readonly", 0, 0, 0);
+  // Create the `search` ACL command category
+  RedisModuleString* cat_name = RedisModule_CreateString(ctx, "search", strlen("search"));
```

Comment:
Maybe wrap this in a function that accepts a `const char*`?

Reply:
What do you mean by "this"? The registration of the new category?

Reply:
Why?
We do this only once, here, and under no circumstances will this code be called elsewhere - that is not aloud.
If it's to save 4 lines of code from this function I prefer not to.